content/financial-aid.html

```diff
@@ -3,6 +3,8 @@
 title: Financial Aid
 ---
 
+<iframe src="https://docs.google.com/forms/d/1JWdkK4pP-USm3duQ-mXdQ6rYEp05lrmqMQ0sSCA5JXw/viewform?embedded=true" width="760" height="800" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+
```

Comment:
Oh, I see, not quite what I meant. I think a link might work better: iframes are a bit gross. =P